### PR TITLE
enhancement: Logo on Navbar implemented

### LINKED ISSUE #15: Web Client Navbar
As a fellow web browser, I want a link to be able to return to the landing page so that I can continue browsing more posts.

navbar for the landing page with the posts
Signin, User, Search, Create Post. (functionality pending)

Acceptance Criteria:
- When the user loads the page a static bar is displayed at the top of the application view.
- When the user scrolls through content the nav bar stays at the top of the browser window.
- When a user clicks the application logo in the left most part of the nav bar they are taken to the "homepage" (posts view)

Task List:
- [x] Create a Navbar react component using Material UI components. /Ari
- [x] Add the site logo as a button on the left most side of the nav bar that when clicks takes a user to the "homepage" (posts view) /Cyrus & Wyatt

### PR DESCRIPTION
**Pull request by ckhorra1**
Wyatt ,Ari, and  Cyrus both imported the icon logo and made it as button. Once the button is clicked it will take the user to the landing page. The navbar will follow to any page it goes to as well as it will still stay at the top while the user scrolls down a page. issue #15

### CODE REVIEW

**Review by zxkevin — CHANGES_REQUESTED** [review]
Though not in AC, please add onClick for the Create Post button to just navigate to /CreatePost even if the page is empty

Nitpick: change the index.html file so that our page title isn't React App anymore

File: packages/hackmcx-client/src/CreatePost.js [hunk 1]
```diff
@@ -0,0 +1,7 @@
+import React from 'react';
```
Comment (zxkevin) [inline packages/hackmcx-client/src/CreatePost.js]: Don't need this file. Please remove

File: packages/hackmcx-client/src/App.js [hunk 2]
```diff
@@ -8,6 +9,7 @@ function App() {
 			<Router>
 				<Routes>
 					<Route exact path='/' exact element={<LandingPage />} />
+					<Route exact path='/CreatePost' exact element={<CreatePost />} />
```
Comment (zxkevin) [inline packages/hackmcx-client/src/App.js]: Didn't need this, but why not

File: packages/hackmcx-client/src/LandingPage.js [hunk 3]
```diff
@@ -40,7 +40,13 @@ const LandingPage = () => {
 							<Button variant='contained'>Search</Button>
 						</Grid>
 						<Grid xs={3}>
-							<Button variant='contained'>Create Post</Button>
+							<Button
+								variant='contained'
+								onClick={() => {
+									navigate('/CreatePost');
```
Comment (zxkevin) [inline packages/hackmcx-client/src/LandingPage.js:46]: This looks good!

**Review by zxkevin — CHANGES_REQUESTED** [review]
Check comments on code

**Review by zxkevin — APPROVED** [review]
This looks good!

**Comment by ckhorra1** [comment]
Thanks for  the reviews!

**Pull request merged**